DS/DS6/EmployeeSortingByName.java

```diff
@@ -0,0 +1,36 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+public class EmployeeSortingByName implements Comparable<Object>{
```

Comment:
ACTION: As discussed before Class Name generally doesn't contain verb, This sounds more like function name than class name.